infra/core/host/appservice.bicep

```diff
@@ -36,33 +38,69 @@ param scmDoBuildDuringDeployment bool = false
 param use32BitWorkerProcess bool = false
 param ftpsState string = 'FtpsOnly'
 param healthCheckPath string = ''
+param allowInboundNetworkRange string = ''
+
+var coreConfig = {
+  linuxFxVersion: linuxFxVersion
+  alwaysOn: alwaysOn
+  ftpsState: ftpsState
+  appCommandLine: appCommandLine
+  numberOfWorkers: numberOfWorkers != -1 ? numberOfWorkers : null
+  minimumElasticInstanceCount: minimumElasticInstanceCount != -1 ? minimumElasticInstanceCount : null
+  minTlsVersion: '1.2'
+  use32BitWorkerProcess: use32BitWorkerProcess
+  functionAppScaleLimit: functionAppScaleLimit != -1 ? functionAppScaleLimit : null
+  healthCheckPath: healthCheckPath
+  cors: {
+    allowedOrigins: union([ 'https://portal.azure.com', 'https://ms.portal.azure.com' ], allowedOrigins)
+  }
+}
+
+
+var coreConfigWithNetworkRules = union(
+  !empty(allowInboundNetworkRange) ? {
+    ipSecurityRestrictions: [
+        {
+            ipAddress: '${allowInboundNetworkRange}/32'
```

Comment:
Will this work if user opts for the CIDR format?

Reply:
Nope, good call-out. Changing to:

ipAddress: contains(allowInboundNetworkRange, '/') ? allowInboundNetworkRange : '${allowInboundNetworkRange}/32'

There's also a parseCidr function in Bicep but I think a simple contains is sufficient?